llvm/test/Analysis/CostModel/X86/arith-fp-codesize.ll

```diff
@@ -1,5 +1,4 @@
 ; NOTE: Assertions have been autogenerated by utils/update_analyze_test_checks.py
-; RUN: opt < %s -enable-no-nans-fp-math  -passes="print<cost-model>" 2>&1 -disable-output -cost-kind=code-size -mtriple=x86_64-- -mattr=-sse2 | FileCheck %s --check-prefixes=CHECK,SSE1
```

Comment:
Instead of dropping the test coverage - can you alter the triple to a suitable i686 target?

Reply:
Good idea!